app/order/order.stache

```diff
@@ -73,7 +73,7 @@
             {{#if saveStatus.isPending}}
               <div class="loading"></div>
             {{else}}
-              <button type="submit" class="btn">Place My Order!</button>
+              <button type="submit" class="btn" {{^ order.items.length}}disabled{{/if}}>Place My Order!</button>
```

Comment:
@chasenlehara you may want to change this to `{{^if order.items.length}}...{{/if}}`.